tests/behat/features/operate-subsites.feature

```diff
@@ -65,10 +68,11 @@ Feature: Create and select a subsite
     Then I should see "file1"
 
     # Add a file to the subsite
-    When I go to "admin/assets"
-    And I select "My subsite" from "SubsitesSelect"
+    When I select "My subsite" from "SubsitesSelect"
+    And I go to "admin/assets"
```

Comment:
Is this change for the same reason as the one with "Ensure that the new subsite is located before adding folders" above?

Reply:
I think so

Reply:
Tried reverting this locally and it passed. Please revert this as it doesn't seem to be necessary (keep the "wait for 2 seconds" below, which does seem to be necessary)